homeassistant/components/airtouch5/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/airtouch5",
   "iot_class": "local_push",
   "loggers": ["airtouch5py"],
-  "requirements": ["airtouch5py==0.2.10"]
+  "requirements": ["airtouch5py==0.2.11"]
```

Comment:
As from the PR description please make a link to the release, change log or a diff in the PR description

Reply:
Sorry, I linked the single commit at the bottom, here is a proper release which I should have done 😓

https://github.com/danzel/airtouch5py/releases/tag/0.2.11
https://github.com/danzel/airtouch5py/compare/0.2.10...0.2.11